.github/workflows/serverless-pre-release-qa.yml

```diff
@@ -26,6 +26,27 @@ on:
       title:
         description: Release title - The title that will be given to this pre-release.
         required: true
+      helpline_code:
```

Comment:
We don't need extra options on the workflow, we just need to pass `environment: development` to the call to `custom_helpline.yml` on line 35

Reply:
@stephenhand  - Oh, I see. Sorry for my misunderstanding. It's done.

Reply:
@stephenhand  - Oh, I see. Sorry for my misunderstanding. It's done.